### PR TITLE
Update UG

### PR DESCRIPTION
**Pull request by Jun-How**
- Added a new section on short tutorial
- Added "stories" to each command as engagement
- Improved the language
- Incorporated and standardised "You" language
- Made miscellaneous and formatting changes
- Added Help command

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -132,39 +141,59 @@ each sub category when sorted.
 
 <div style="page-break-after: always;"></div>
 
-## 4. Features <a name="features"></a>
+## 4. Short Tutorial <a name="tutorial"></a>
+
+Excited to try **MODTrek** out? Here is a sequence of basic commands to help you start tracking your modules:
+
+1. `add /m CS1101S /c 4 /y Y1S1`: Adds the module CS1101S to the list. The Results Section should display the Module List, with a Module Card created for CS1101S, placed within a Module Group categorised as Y1S1 (sorted by year-sem by default).
```
Comment (jmestxr) [inline docs/UserGuide.md:148]: I have discussed with @Cheamybunny and we have decided that there's no need to introduce the ModuleList, ModuleGroup and ModuleCard components in the UG as they are intuitive enough to the user using the GUI. It is originally only meant for the DG to connect each UI component to its corresponding Java class.

In UG, we will just introduce the three screens of the Results Section: Degree Progress Screen, Module List Screen,  Module Search Screen. @Cheamybunny will reflect this change in a future commit.

So this part can be updated to reflect this change.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -110,20 +117,22 @@ The following is an annotated breakdown of **MODTrek's** GUI:
...
 The following is the annotated breakdown of the Results Section of **MODTrek's** GUI:
 ![UI subsection](images/Ui-subsections-detailed.png)
 <p align="center"><em>Breakdown of Results Section</em><br></p>
 
-1. **Doughnut Chart:** This will be shown on the degree progress section. It shows the completion progress of the degree
-and individual sub requirements of the degree. Your total MCS completed and CAP are also displayed.
-2. **Module List:** This will be shown in both the module list section and module search section. It shows all the module in the list.
-3. **Module Group:** This will be shown in both the module list section and module search section. It shows the modules in
-each sub category when sorted.
+1. **Doughnut Chart:** This will be shown in the degree progress section. It shows your completion progress of the degree
```
Comment (jmestxr) [inline docs/UserGuide.md:131]: I have discussed with @Cheamybunny and we have decided that there's no need to introduce the ModuleList, ModuleGroup and ModuleCard components in the UG as they are intuitive enough to the user using the GUI. It is originally only meant for the DG to connect each UI component to its corresponding Java class.

In UG, we will just introduce the three screens of the Results Section: Degree Progress Screen, Module List Screen, Module Search Screen. @Cheamybunny will reflect this change in a future commit.

Comment (Cheamybunny) [reply docs/UserGuide.md:131]: I will do this tyvm

**Review by Cheamybunny — APPROVED** [review]
Lovely

**Pull request merged**